SEL32/sel32_con.c

```diff
@@ -259,43 +343,16 @@ t_stat con_srvo(UNIT *uptr) {
 
     switch (cmd) {
 
-    /* if input tried from output device, error */
-    case CON_RD:        /* 0x02 */          /* Read command */
-    case CON_ECHO:      /* 0x0a */          /* Read command w/ECHO */
-    case CON_RDBWD:     /* 0x0c */          /* Read Backward */
-        /* if input requested for output device, give error */
-        uptr->SNS |= SNS_CMDREJ;            /* command rejected */
-        uptr->CMD &= LMASK;                 /* nothing left, command complete */
-        sim_debug(DEBUG_CMD, dptr,
-            "con_srvo Read to output device CMD %08x chsa %04x cmd = %02x\n", uptr->CMD, chsa, cmd);
-        chan_end(chsa, SNS_CHNEND|SNS_UNITCHK); /* unit check */
-        break;
-
-    case CON_CON:       /* 0x1f */          /* Connect, return Data Set ready */
-        uptr->SNS |= (SNS_DSR|SNS_DCD);     /* Data set ready, Data Carrier detected */
-        sim_debug(DEBUG_CMD, dptr,
-            "con_srvo CON CMD %08x chsa %04x cmd = %02x\n", uptr->CMD, chsa, cmd);
-        uptr->CMD &= ~CON_MSK;              /* remove old CMD */
-        chan_end(chsa, SNS_CHNEND|SNS_DEVEND);  /* return OK */
-        break;
-
-    case CON_DIS:       /* 0x23 */          /* Disconnect has do nothing */
-        uptr->SNS &= ~(SNS_DSR|SNS_DCD);    /* Data set not ready */
-        sim_debug(DEBUG_CMD, dptr,
-            "con_srvo DIS CMD %08x chsa %04x cmd = %02x\n", uptr->CMD, chsa, cmd);
-        uptr->CMD &= ~CON_MSK;              /* remove old CMD */
-        chan_end(chsa, SNS_CHNEND|SNS_DEVEND);  /* return OK */
-        break;
-
     case CON_INCH2:      /* 0xf0 */         /* INCH command */
         uptr->CMD &= LMASK;                 /* nothing left, command complete */
         sim_debug(DEBUG_CMD, dptr,
-            "con_srvo INCH unit %02x: CMD %08x cmd %02x incnt %02x u4 %02x\n",
-            unit, uptr->CMD, cmd, con_data[unit].incnt, uptr->u4);
+            "con_srvo INCH unit %02x: CMD %08x cmd %02x incnt %02x u4 %02x inch %06x in2 %06x\n",
+            unit, uptr->CMD, cmd, con_data[unit].incnt, uptr->u4, mema, M[mema>>2]);
 
         /* now call set_inch() function to write and test inch buffer addresses */
         /* 1-256 wd buffer is provided for 128 status dbl words */
-        tstart = set_inch(uptr, mema, 128); /* new address & 128 entries */
+//TRY   tstart = set_inch(uptr, mema, 128); /* new address & 128 entries */

```

Comment:
What is the purpose of this "TRY" line?  A note to you to try this in the future?  Leftover no longer needed code?

Reply:
Try is again a debugging note to myself.  Will usually go away the next cleanup cycle.